client/app/components/Modal/Modal.scss

```diff
@@ -52,7 +52,9 @@
       }
       &Close {
         cursor: pointer;
-        margin-left: auto;
```

Comment:
Let's not use position: absolute here. It's not necessary.

Reply:
![without](https://user-images.githubusercontent.com/16218249/48751925-26f01680-ec44-11e8-8540-f346fa53616c.png)
Without position absolute, the close button takes up 15px at the top of the modal _as if there were a title_ there. With position absolute, I can have the close button at the top-right without pushing the rest of the content down since I'm _not_ using the title prop of the modal component here.